README.md

```diff
@@ -2,12 +2,16 @@
 The operator-controller is the central component of Operator Lifecycle Manager (OLM) v1. It extends Kubernetes with an API through which users can install Operators.
 
 ## Description
-OLM v1 is the follow-up to OLM v0, located [here](https://github.com/operator-framework/operator-lifecycle-manager). It consists of four different components, including this one, which are as follows:
+OLM v1 is the follow-up to [OLM v0](https://github.com/operator-framework/operator-lifecycle-manager). The vision for OLM V1 is discussed in detail in the [OLM V1 Product Requirements Document](https://docs.google.com/document/d/1-vsZ2dAODNfoHb7Nf0fbYeKDF7DUqEzS9HqgeMCvbDs/edit). 
+
+The Operator Controller project consists of four different components, including this one, which are as follows:
 * operator-controller
 * [deppy](https://github.com/operator-framework/deppy)
 * [rukpak](https://github.com/operator-framework/rukpak)
 * [catalogd](https://github.com/operator-framework/catalogd)
 
+Ongoing feature development can always be tracked at the [OLM V1 GitHub Project](https://github.com/orgs/operator-framework/projects/8) by looking at `Milestone #` tabs. If you are interested in getting involved, please see the [CONTRIBUTING.md](TO/Be/Added) for more information.
```

Comment:
Nit: IMO there also should be a `Contributing` section in the Readme. It can simply include the same line that is here:
```
If you are interested in getting involved, please see the [CONTRIBUTING.md](TO/Be/Added) for more information.
```
This just adds a bit of additional discoverability on how to contribute

Reply:
Turns out that the doc already had a contributing section so I moved that line there.